### PR TITLE
Integrate CONNECT ID

### PR DESCRIPTION
**Pull request by jorunfa**
TODO: Change client to telenordigital-protoman-android once available.

### CODE REVIEW

File: app/src/main/java/com/telenordigital/protoman/MainActivity.kt [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+        val loginButton = findViewById<ConnectLoginButton>(R.id.login_button)
+        loginButton.setLoginScopeTokens("profile openid")
+
+        ConnectSdk.handleRedirectUriCallIfPresent(intent, object : ConnectCallback {
+            override fun onSuccess(successData: Any) {
+                goToSignedInActivity()
+            }
+
+            override fun onError(errorData: Any) {
+                Toast.makeText(this@MainActivity, "Failed to sign in", Toast.LENGTH_SHORT).show()
```
Comment (metareven) [inline app/src/main/java/com/telenordigital/protoman/MainActivity.kt:27]: Should we not log the error somewhere as well?

**Review by StephenLC — COMMENTED** [review]
I can read

**Review by StephenLC — COMMENTED** [review]
LGTM

**Review by metareven — APPROVED** [review]
Looks OK. This is just a simple prototype after all

**Pull request merged**